app/features/build/build_websocket_backend.rb

```diff
@@ -48,9 +38,9 @@ def call(env)
       ws.on(:open) do |event|
         logger.debug([:open, ws.object_id])
 
-        url_details = fetch_build_details(event)
-        build_number = url_details[:build_number]
-        project_id = url_details[:project_id]
+        params = Rack::Request.new(env).params
+        build_number = params["build"].to_i
+        project_id = params["project"]
```

Comment:
Let's be consistent with naming things, `project` and `project_id`, e.g.: `params["project"]` -> `params["project_id"]`
same with `build` and `build_number`.

Reply:
fixed